packages/experimental/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,99 @@
+/** @jsx withSlots */
+import { useState, useEffect, useMemo } from "react";
+import { Circle, ClipPath, Defs, LinearGradient, Rect, Stop, Svg, Image } from 'react-native-svg';
+import { shimmerName, ShimmerProps, ShimmerType } from './Shimmer.types';
+import { mergeProps } from '../../../framework/merge-props/lib/mergeProps';
+import { buildUseStyling, compose, UseSlots, withSlots } from '../../../experimental/framework/lib';
+import { Animated } from 'react-native';
+import { stylingSettings } from './Shimmer.styling';
+
+export function renderAnimations(memoData: any) {
+  let startValue = useState(new Animated.Value(0))[0]
+  useEffect(() => {
+  Animated.loop(
+    Animated.sequence([
+    Animated.timing(startValue, {
+      toValue: memoData.toValue,
+      duration: memoData.duration,
+      delay: memoData.delay,
+      useNativeDriver:false
+    })]),
+  ).start()
+  })
+  return startValue;
+}
```

Comment:
I think you've in effect made a custom hook here? If so, you might want to rename this to something like "useShimmerAnimation" to reflect that. https://reactjs.org/docs/hooks-custom.html

Reply:
yep, renamed